mock/reader.go

```diff
@@ -5,33 +5,34 @@ import (
 
 	"github.com/influxdata/flux/memory"
 	"github.com/influxdata/influxdb/v2/query/stdlib/influxdata/influxdb"
+	"github.com/influxdata/influxdb/v2/storage/reads"
 )
 
-type StoreReader struct {
+type Reader struct {
```

Comment:
Does this name need to be changed? The name `Reader` is a very generic name for a package and since this is a mock package the intention was to say what kind of reader this was. It's a mock of reading the storage engine. If you want, we can name it `StorageReader` to match the changed interface name that is on the algorithm w branch.

Reply:
OK `StorageReader` sounds good.